spec/serializers/topic_list_item_serializer_spec.rb

```diff
@@ -207,4 +207,28 @@
       DiscoursePluginRegistry.unregister_modifier(plugin, modifier, &proc)
     end
   end
+
+  describe "is_hot" do
```

Comment:
```suggestion
  describe "#is_hot" do
```

Minor but an attribute in a serializer is essentially an instance method.